### ISSUE TITLE
Excel file name with ( ) failed.

### ISSUE THREAD

**Issue by LauKL1991** [nolze/msoffcrypto-tool#25]
Excel file name with (), for example, test(1).xlsx will always return no password protection (even password protected).

**Comment by nolze** [comment]
Hi, I tried myself but could not reproduce the bug.

I renamed `example_password.xlsx` [here](https://github.com/nolze/msoffcrypto-tool/tree/master/tests) to `example_password(1).xlsx` and ran the command successfully with the following output.

```
$ msoffcrypto-tool --test -v example_password\(1\).xlsx
Version: 4.6.4
OOXMLFile.type: agile
example_password(1).xlsx: encrypted
```

Please make sure that you escape or quote the file path/name to avoid the conflict with your shell (eg. `example_password\(1\).xlsx` or `'example_password(1).xlsx'`). Or I'll investigate the file if you could provide it with or without password.

**Comment by LauKL1991** [comment]
Hello,

Thanks for the reply. I attach the files for you to test. (both file password is _2019BDRP_jan_)
For file encrypted1.xlsx it worked well. However, the encrypted1(1).xlsx prompted it's not encrypted. 
I did escape the quote _uploads/process/encrypted1\(1\).xlsx_, but it still doesn't work. Also, it is the latest version: 4.6.4

[encrypted1(1).xlsx](https://github.com/nolze/msoffcrypto-tool/files/2986286/encrypted1.1.xlsx)
[encrypted1.xlsx](https://github.com/nolze/msoffcrypto-tool/files/2986287/encrypted1.xlsx)

**Comment by nolze** [comment]
Thanks! Though I tested your files, I could check their encryption status correctly and decrypt them without error this time too.

As far as I investigated the files, there is nothing wrong with them. And as far as I know, the program does nothing special with regard to the file name (hopefully). So the problem may have something to do with the system-specific matters (OS, terminal, shell, text encoding, etc.).

For your information, I tried testing your files with Python 3.7 on macOS 10.13 and on Arch Linux. I haven't tested it on Windows (in case that you're using Windows).
